packages/server-boilerplate/src/microService/api/ApiBuilder.ts

```diff
@@ -27,7 +29,7 @@ export class ApiBuilder {
     this.models = new ModelRegistry(transactingConnection);
     this.app = express();
 
-    this.version = 'v1'; // Default version
+    this.version = 'v[0-9]'; // Default version
```

Comment:
Out of curiosity, what is going on here?

Reply:
Ahh good question! So basically our api endpoints always go `http://<host>/<version>/<endpoint>` and so the ApiBuilder allows you to set a version so that it can staple it to each route, thus saving a us from having to remember to put it in when defining every url. 

The problem is that for some reason some of the `.env` vars preset in our deployment instances used `v2` for report-server... I considered changing them but kinda figured that unless we are explicitly using versioning (which I don't think any of our services actually do) then why not just make it a wildcard to make life easier?